docs/_conversion/overview.md

```diff
@@ -21,3 +21,5 @@ The tool will guide you through a three step process:
 1. Update your existing users’ Zaps to use the new integration. 
 
 The legacy platform will be disabled this year, so this process is required to maintain your integration. We’ve worked hard to make this process as easy as possible while avoiding any impact on users.
+
+>**This feature is being progressively rolled out, and may not be immediately available for your integration. It is planned to be available to everyone within a few weeks.**
```

Comment:
Rather than "a few weeks", I'd put "by April 1 2021" so readers will know if the doc is stale

Reply:
Changed.  I was on the fence as I wrote that, and you're right.